android/src/main/java/io/customer/reactnative/sdk/messagingpush/RNCIOPushMessaging.kt

```diff
@@ -77,27 +76,27 @@ class RNCIOPushMessaging(
     }
 
     /**
-     * Handles push notification received. This can be helpful in processing push notifications
+     * Handles push notification received. This is helpful in processing push notifications
      * received outside the CIO SDK.
      *
      * @param message push payload received from FCM.
+     * @param handleNotificationTrigger indicating if the local notification should be triggered.
```

Comment:
I don't understand the use of this parameter.

Reply:
Levi, it is there for the same reason it's in [Android SDK](https://github.com/customerio/customerio-android/blob/main/messagingpush/src/main/java/io/customer/messagingpush/CustomerIOPushNotificationHandler.kt#L83). There was a thread shared on slack along with the flowchart that you saw and commented on, it's there to exactly cater to that case. 

To avoid *duplication* we need to ask Customers to display notifications themselves for a simple push? and if they do that how do we expect them to track notifications? We expect them to still use this method but with the `handleNotificationTrigger ` as `false` so we only track metric for them and not generate the push notification and they don't have to manually extract `device id` and `delivery id` and then manually track it, keeping it consistent to native as well. 

You can argue that commenting on it could be better, but customers would understand it when we make it as part of our documentation along with the flow chart of where this would be helpful to them.

Reply:
Because the parameter exists in the native Android SDK today and this parameter is to add the feature to the RN SDK, I approve.